### PR TITLE
Add a installing wine guide.

### PR DESCRIPTION
**Pull request by PetrifiedOak**

### CODE REVIEW

File: src/Installation/wine.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Installing wine
+
+A wine version higher than 7 is required, 8.3+ is recommended.
```
Comment (apprehensions) [inline src/Installation/wine.md:3]: Provide a reasoning on why 8.3 is 'reccomended'

File: src/Installation/wine.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+You can see your wine version with `wine --version`
```
Comment (apprehensions) [inline src/Installation/wine.md:4]: This may not work on flatpak, `vinegar exec --version`

File: src/Installation/wine.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+### Debian based distrobutions.
+
+See [winehq's guide](https://wiki.winehq.org/Ubuntu).
```
Comment (apprehensions) [inline src/Installation/wine.md:8]:
```suggestion
See [WineHQ's guide](https://wiki.winehq.org/Ubuntu).
```

File: src/Installation/wine.md (hunk 1 above)
Comment (PetrifiedOak) [reply src/Installation/wine.md:3]: Low Fragmentation Heap, without it heavier games lag

File: src/Installation/wine.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Installing wine
+
+A wine version higher than 7 is required, 8.3+ is recommended.
+You can see your wine version with `wine --version`
+
+### Debian based distrobutions.
```
Comment (apprehensions) [inline src/Installation/wine.md:6]: its `distributions`.

File: src/Installation/wine.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+### Arch based distrobutions.
+
+First, make sure to add the multilib repository and the community repository. [Official repositories](https://wiki.archlinux.org/title/Official_repositories)
+
+After that it should just be as simple as a `pacman -S wine`
+
+### Fedora
+
+See [winehq's guide](https://wiki.winehq.org/Fedora)
```
Comment (apprehensions) [inline src/Installation/wine.md:18]: same thing

File: src/Installation/wine.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+### Debian based distrobutions.
+
+See [winehq's guide](https://wiki.winehq.org/Ubuntu).
+
+### Arch based distrobutions.
+
+First, make sure to add the multilib repository and the community repository. [Official repositories](https://wiki.archlinux.org/title/Official_repositories)
+
+After that it should just be as simple as a `pacman -S wine`
```
Comment (apprehensions) [inline src/Installation/wine.md:14]: This documentation is extremely self deprecating and third person.

Comment (PetrifiedOak) [reply src/Installation/wine.md:14]: What should i add otherwise?

File: src/Installation/wine.md (hunk 2 above)
Comment (PetrifiedOak) [reply src/Installation/wine.md:4]: Thanks for the input, im adding everything right now

File: src/Installation/wine.md (hunk 1 above)
Comment (apprehensions) [reply src/Installation/wine.md:3]: ON the wiki Oak.

Comment (PetrifiedOak) [reply src/Installation/wine.md:3]: https://www.winehq.org/news/2023030301
Low Fragmentation Heap got implemented, if youve played a bit heavier games on wine without LFH you would know.

**Comment by apprehensions** [comment]
superceded by 169451ed2789c3e5dafa7bc30d05939a8378441c

**Pull request closed without merging**